osf/management/commands/import_EGAP.py

```diff
@@ -119,11 +122,12 @@ def get_egap_assets(guid, creator_auth):
 
     with ZipFile(egap_assets_path, 'r') as zipObj:
         zipObj.extractall(temp_path)
-        zip_parent = [file for file in os.listdir(temp_path) if os.path.isdir(file) and file != '__MACOSX']
+        zip_parent = [file for file in os.listdir(temp_path) if file not in ('__MACOSX', 'egap_assets.zip') and not check_id(file)]
```

Comment:
This seems hacky. `zipfile` should let you extract into a specific path, so you'll know the name, or you should be able to get the name of the directory that it's going to extract into. Could you take a few minutes to see about finding a better way to do this?

Reply:
It's not the whole zip file content I'm moving it's everything under the parent and since there no way of knowing what the parent is named, there's no way of moving the children other then selecting the everything that doesn't have an id and isn't `__MACOSX`. 

That line is actually just there to pass tests because tests. The test zip doesn't have a parent directory it's just a flat bundle of folders, if the zips used were zipped flatly like that, we wouldn't need any additional code.

Reply:
Sounds good. Did you get this to work locally with the test file?

Reply:
Yep